language/diem-tools/oncall-trainer/src/main.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use move_oncall_trainer::{experiments, run_experiment, NodeInfo};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct Opt {
+    /// Path to the output serialized bytes
+    #[structopt(long, short)]
+    experiment: String,
+}
+
+fn main() -> Result<()> {
+    let opt = Opt::from_args();
+
+    let node = NodeInfo::new_local();
+    println!("A Diem Validator node has been spawned! Here's the info you need:");
+    println!("{}", node);
+
+    println!("Here's the command to start a few tools that might be handy for you:");
+    println!(
+        "cargo run --bin diem-transaction-replay -- --url {}",
+        node.json_rpc
+    );
+    println!("cargo run --bin cli -- --url {}", node.json_rpc);
+    println!(
+        "cd testsuite/cli; cargo run -- --chain-id {} --url {} --waypoint {} -m {:?}",
```

Comment:
What would you think about changing this command to a `cargo run --bin cli` etc. command here instead of having them `cd` to the directory?

Also, is the rootkey path absolute? If not it may be a good idea to make it absolute so that users don't need to worry about which directory they're currently in.

Reply:
> What would you think about changing this command to a cargo run --bin cli etc. command here instead of having them cd to the directory?

Unfortuately cli is not a binary available at top level. That's why the cd is a must.

> Also, is the rootkey path absolute? 

It is indeed an absolute path.